pagopa/pn-frontend
# Digital domicile - feature pn-12391

## Short description
Implemented digital domicile feature

## How to test
Login with pf/pg -> go to Recapiti page -> check that digital domicile feature is ok
